floor/config/layouts/one-pod-layout.json

```diff
@@ -1,5 +1,5 @@
 {
-  "squares": 64,
+  "squares": 8,
   "rows": 8,
   "cols": 8,
```

Comment:
Squares is now meaningful, but rows and cols still aren't (everything still assumes an 8x8 square layout).